src/const.ts

```diff
@@ -120,6 +120,16 @@ export interface BotStudioEditProps {
   suggestedRepliesDirection?: 'horizontal' | 'vertical';
 }
 
+export interface MessageInputControlProps {
```

Comment:
Explicit naming is good, but it would be better to minimize parts that can be quite clearly inferred implicitly. For instance, "message input" could be simplified to just "input"

Reply:
oh input could mean many things so I thought necessary to use messageinput here. But since many of you think otherwise, I will remove it!